apps/maestro/src/config/evm-chains.ts

```diff
@@ -76,6 +76,45 @@ export const ALL_CHAINS: ExtendedWagmiChainConfig[] = [
     axelarChainName: "ethereum",
     environment: ENVIRONMENTS.mainnet,
   },
+  // TODO: correct this
+  {
+    id: 101,
```

Comment:
since Sui isn't an EVM chain, we should either create a separate sui config file or rename thsi file